std_msgs/Header header
int64 casualty_id

sensor_msgs/NavSatFix gps
float64 easting
float64 northing

geometry_msgs/Point local
geometry_msgs/Point relative
geometry_msgs/Point pixel

sensor_msgs/CompressedImage image
